FunctionalPreprocessing/SubmitFunctionalPreprocessingCheckHPC

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
This file is part of the changes for being able to check prerequisites, submit jobs, and check completion status from a "local" machine instead of having to be logged in to the CHPC cluster to do those things. Right?

This is the script that is invoked across computers using SSH from the local machine to the CHPC machine. Right? 

If that's right, let's move this file into a different branch for those changes (maybe feature/remote_invocation) and remove them from this branch and this pull request. I'd prefer that when the set of changes associated with this change request are merged, we can merge them all in under a comment of "Change Location of Running Status Files to put them in a Database Resource".

Reply:
Yes, following 4 files,
SubmitFunctionalPreprocessingCheckHPC
SubmitFunctionalPreprocessingJobSubmitHPC
SubmitFunctionalPreprocessingPrereqHPC
SubmitFunctionalPreprocessingStatusHPC